src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * Enums and configuration object mapping.
+ */
+// Colors taken from clinicalAttributes.scss
+export const colors = {
+    sample_color_primary: 'black',
+    sample_color_recurrence: 'orange',
+    sample_color_metastasis: 'red',
+    sample_color_cfdna: 'blue',
+    sample_color_xenograft: 'pink',
+};
+
+/**
+ * Object mapping of keys to objects indicating
+ * whether an attribute is primary or secondary,
+ * the formatter to display the inputted value t,
+ * and font coloring based on the key and value.
+ */
+export const config = {
+    AGE: {
+        primary: true,
+        formatter: t => {
+            return `${t} years old`;
+        },
+        order: 2,
+    },
+    BREAST_CANCER_RECEPTOR_STATUS: {
+        order: 6,
+    },
+    CANCER_TYPE: {
+        primary: true,
+        order: 3,
+    },
+    CANCER_TYPE_DETAILED: {
+        primary: false,
+        order: 4,
+    },
+    DERIVED_NORMALIZED_CASE_TYPE: {
+        primary: true,
+        values: {
+            Primary: colors.sample_color_primary,
+            Progressed: colors.sample_color_recurrence,
+            Recurrence: colors.sample_color_recurrence,
+            Metastasis: colors.sample_color_metastasis,
+        },
+        order: 1,
+    },
+    DERIVED_SAMPLE_LOCATION: {
+        primary: false,
+        order: 2,
+    },
+    DFS_STATUS: {
+        primary: true,
+        values: {
+            _: '#f00',
+            DiseaseFree: 'rgb(0, 128, 0)',
+            Yes: 'rgb(0, 128, 0)',
+        },
+        order: 9,
+    },
+    DFS_MONTHS: {
+        primary: false,
+        formatter: t => {
+            return `${t} months`;
+        },
+        order: 10,
+    },
+    DRIVER_MUTATIONS: {
+        order: 6,
+    },
+    ERG_FUSION_ACGH: {
+        primary: true,
+        formatter: t => {
+            return `ERG-fusion aCGH: ${t}`;
+        },
+        order: 6,
+    },
+    ETS_RAF_SPINK1_STATUS: {
+        order: 6,
+    },
+    GLEASON: {
+        primary: true,
+        formatter: t => {
+            return `Gleason: ${t}`;
+        },
+    },
+    GLEASON_SCORE: {
+        order: 6,
+    },
+    GENDER: {
+        order: 1,
+    },
+    HISTOLOGY: {
+        order: 6,
+    },
+    KNOWN_MOLECULAR_CLASSIFIER: {
+        order: 5,
```

Comment:
Why is `primary` an optional field? Is `primary: undefined` different than `primary: false`?

Reply:
I forgot to specify `primary` which is what led to the parentheses wrapping around the GENDER and SEX attributes, but this has been corrected for the latest commit

Reply:
However wherever primary is undefined, the logic in stringBuilder should return false